### DISCUSSION TITLE
Welcome and Intros!

### DISCUSSION THREAD

**Discussion by RaoOfPhysics** [alan-turing-institute/asg-research-communications#27] [Announcements]
## 👋 Hello and welcome!

We’re using Discussions as a place to connect with other members of the wider ASG Community. We hope that you:

- ask questions you’re wondering about,
- share ideas,
- engage with other community members and
- welcome others and are open-minded.

To get started, comment below with an introduction of yourself and tell us about your projects and interests.

**Comment by RaoOfPhysics** [comment]
Hi all!

I’m [Achintya](https://achintyarao.in), the ASG Community Manager. I have a background in physics and science communication, and am fairly new to the world of AI research.

My project is to nourish this community, and I’m also looking forward to sharing your successes with the world via a variety of communication outputs we’ll produce between now and early 2023.

**Comment by aldenc** [comment]
Hi, I'm Alden, I'm a Research Application Manager 🐏 My background is in neurobiology and microscopy and I too am quite new to AI and data science!

As a RAM I work with ASG-funded projects to help them embed their research outputs in the real world and enagage with external stakeholders who can benefit from the work. If that sounds interesting to you please send me an email or ping me on Slack!
